parser/parser_test.go

```diff
@@ -363,8 +362,6 @@ func (s *testParserSuite) TestBuiltin(c *C) {
 		{`SELECT LOCATE('bar', 'foobarbar');`, true},
 		{`SELECT LOCATE('bar', 'foobarbar', 5);`, true},
 
-		{"select current_date, current_date(), curdate()", true},
```

Comment:
@zimulala Please note these test cases are reorganized with some newly added ones from line 402 - 426. They are listed in an order from small time units to big time units.